src/app/index/index.component.html

```diff
@@ -17,9 +17,7 @@ <h2 class="yacy" id="greeting"></h2>
         <div id="search-bar">
             <app-search-bar></app-search-bar>
         </div>
```

Comment:
There seems to be some indentation error here, along with a few extra blank lines.